tests/testthat/test-dbWriteTable.R

```diff
@@ -15,3 +15,54 @@ test_that("options work", {
   dbWriteTable(con, "dat", "dat-n.txt", sep="|", eol="\n", overwrite = TRUE)
   expect_equal(dbReadTable(con, "dat"), expected)
 })
+
+test_that("appending is error if table does not exist", {
+  if (!mysqlHasDefault()) skip("Test database not available")
+
+  con <- dbConnect(MySQL(), dbname = "test")
```

Comment:
Can you please switch to the new style `con <- mysqlDefault()` (and drop the explicit `skip()`)